app/controllers/signup_controller.rb

```diff
@@ -96,11 +96,15 @@ def profile
                     end
                   end,
                   failure: lambda do
-                    render :profile
+                      Rails.logger.warn '=== Here is your problem ==='
+                      Rails.logger.warn all_errors
+                      Rails.logger.warn '=== end of error listing ==='
+                      render :profile
                   end)
```

Comment:
I think this is leftover dev code, since Rails.logger isn't really used on production. if you want to retain the errors, they should be logged to sentry, right?